lib/phoenix_live_reload/web_console_logger_initializer.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Phoenix.LiveReloader.WebConsoleLoggerInitializer do
```

Comment:
In case Application.stop does not work, let's keep this, but let's merge these functions into WebConsoleLogger itself. :)

Reply:
Done in 0527b4f14945e738c0af41099100703f4f2786e1!